### PR TITLE
docs: add recommendation to install nuxtr

### PR DESCRIPTION
**Pull request by adhamfarrag**
### 🔗 Linked issue
None.

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Adding [Nuxtr](https://marketplace.visualstudio.com/items?itemName=Nuxtr.nuxtr-vscode) extension to addional steps for optimal setup. 

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by atinux — APPROVED** [review]

**Comment by Hebilicious** [comment]
Shouldn't we wait until this extension becomes open source?

**Comment by adhamfarrag** [comment]
We are waiting :)

**Comment by danielroe** [comment]
Marking as draft to help me with triaging PRs. Please do mark as ready whenever we're good to go 🙏

**Comment by Hebilicious** [comment]
@adhamfarrag I think nuxtr would be a really great addition to the ecosystem!
Do you have an ETA for open sourcing your extension? I would be happy to contribute some features.

On a side note, I found out about another Nuxt vscode OSS extension https://marketplace.visualstudio.com/items?itemName=cpreston321.nuxt-vscode

**Comment by adhamfarrag** [comment]
@danielroe just open sourced it.

File: docs/1.getting-started/2.installation.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Start with one of our starters and themes directly by opening [nuxt.new](https:/
 ::details
 :summary[Additional notes for an optimal setup:]
 - **Node.js**: Make sure to use an even numbered version (18, 20, etc)
-
+- **Nuxtr**:   Install Nuxtr extension for Visual Studio Code from [**this URL**](https://marketplace.visualstudio.com/items?itemName=Nuxtr.nuxtr-vscode)
```
Comment (danielroe) [inline docs/1.getting-started/2.installation.md:34]:
```suggestion
- **Nuxtr**: Install the community-developed [Nuxtr extension](https://marketplace.visualstudio.com/items?itemName=Nuxtr.nuxtr-vscode)
```

**Review by danielroe — APPROVED** [review]

**Pull request merged**